packages/mobile/src/config.ts

```diff
@@ -26,8 +26,6 @@ export const CELO_VERIFIER_START_MINING_LINK = 'celo://verifier/start'
 export const CELO_FAUCET_LINK = 'https://celo.org/app'
 export const CELO_TERMS_LINK = 'https://celo.org/terms'
 
-export const DEFAULT_COUNTRY = Config.DEFAULT_COUNTRY || null
```

Comment:
Why remove this? In fact, if anything we should set it to Liberia fro the pilot right?

Reply:
We don't use it anymore anywhere - searching for DEFAULT_COUNTRY doesn't show anything

Reply:
Ah good catch. The phone number input component still supports it though:
packages/react-components/components/PhoneNumberInput.tsx

So should we add it back?